05/DB.test.js

```diff
@@ -1 +1,21 @@
-import DB from './DB';
\ No newline at end of file
+import DB from "./DB";
+describe("DB", () => {
+	describe("Method insert()", () => {
+		it("return error when is not a number", async () => {
+			const db = new DB();
+			const data = { id: "A" };
+			const result = db.insert(data);
+			await result.catch((err) =>
+				expect(err).toBe("ID can be only number!")
+			);
+		});
+		it("return error when ID's are duplicated", async () => {
+			const db = new DB();
+			const data = { id: 1, id: 1 };
+			const result = db.insert(data);
```

Comment:
Jeśli chcemy dodać kolejne elementy to musimy wieloktronie wywoływać `insert` tj.
```
expect.assertions(1);

const db = new DB();
try {
  await db.insert({id: 1});
  await db.insert({id: 1});
} catch(err) {
  expect(err).toBe("ID can't be duplicated!");
};
```
lub
```
expect.assertions(1);

const db = new DB();
return db.insert({id: 1})
  .then(() => db.insert({id: 1}))
  .catch((err) => {
    expect(err).toBe("ID can't be duplicated!");
});
```